guides/common/modules/proc_preparing-a-host-for-external-databases.adoc

```diff
@@ -2,20 +2,20 @@
 = Preparing a Host for External Databases
 
 ifndef::orcharhino[]
-Install a freshly provisioned system with the latest {EL} 7 server to host the external databases.
+Install a freshly provisioned system with the latest {EL} 8 server to host the external databases.
 endif::[]
 ifdef::orcharhino[]
-Install a freshly provisioned system with the latest CentOS 7, Oracle Linux 7, or {RHEL} 7 to host the external databases.
+Install a freshly provisioned system with the latest CentOS 8 Stream, Oracle Linux 8, or {RHEL} 8 to host the external databases.
 endif::[]
 
+.Prerequisites
+* The prepared host must meet {Project}'s {InstallingServerDocURL}storage-requirements_{project-context}[Storage Requirements].
+
 ifdef::satellite[]
-Subscriptions for Red{nbsp}Hat Software Collections and {RHEL} do not provide the correct service level agreement for using {Project} with external databases.
+Subscriptions for {RHEL} do not provide the correct service level agreement for using {Project} with external databases.
 You must also attach a {Project} subscription to the base operating system that you want to use for the external databases.
 endif::[]
 
-.Prerequisites
-* The prepared host must meet {Project}'s {InstallingServerDocURL}storage-requirements_{project-context}[Storage Requirements].
```

Comment:
There are no more storage requirements?

Reply:
Not on this line there isn't! See about 5 lines up. I think these got swapped around when I had to put something back. This will get adjusted in the next push.